### DISCUSSION TITLE
How to uninstall a moudle without root or other root method?

### DISCUSSION THREAD

**Discussion by rm52** [bmax121/APatch#363] [Q&A] [answered]
I installed a moudle which causes bootloop.
I flash original boot img or mask's boot img, it stop bootloop.
But it bootloop again with apatch's boot img.
How to uninstall an apatch's moudle without root or other root img?

**Comment by marat2509** [comment, marked as answer]
You can mount /data/adb/ap/modules.img, and from the mounted image you can delete the module you need. 

Another option, but more crude, is to delete or rename this file, which will unmount all modules at boot time

Also you can press Volume- about 3 times on booting
